### ISSUE TITLE
Project-local `keymap.lisp` being ignored

### ISSUE THREAD

**Issue by fosskers** [Shirakumo/trial#81]
Hi there, thank you for Trial. I'm trying out the Quickstart guide and have managed to get the spinning cat box to move around. The movement, however, required that I dig into [`load-keymap`](https://github.com/Shirakumo/trial/blob/master/settings.lisp#L42-L54), and after some debugger stepping, noticed that:

1. The guide tells us to call `(load-keymap)` as-is, which defaults the `path` argument to `(keymap-path)`. For me, this resolves to `/home/colin/.config/shirakumo/save-the-farm/keymap.lisp`, not the file in my local project.
2. The `default` value calculated next _does_ resolve to the file I expected: `/home/colin/code/common-lisp/save-the-farm/keymap.lisp`.
3. The initial branch of the `cond` doesn't pass when there is only a `default` and no global keymap.
4. Thus, the `T` branch is triggered, but passes `path` to be read, not `default`.
5. No keymappings actually get loaded.
6. Kitty doesn't move.

I fixed this by patching (CL is awesome) the `T` branch to call `(load-mapping default :package package)` instead, and kitty moves. 

The `or` is perhaps missing a condition where only `default` exists. Thoughts?

**Comment by Shinmera** [comment]
Huh? The first branch should trigger since `probe-file` for the path will return `NIL`?

**Comment by fosskers** [comment]
I started another round of step debugging but then decided to actually look into `~/.config/shirakumo/save-the-farm/`. Sure enough there was a `keymap.lisp`, but it was empty and I hadn't created it.

Perhaps this happened because at first I placed `keymap.lisp` in `src/`, not at the project root, the first time I tried to test the bindings. Trial, not seeing a local keymap where it expected one, perhaps generated an empty one and defaulted to that even after I moved my own to the project root.

I've reverted my local changes, deleted the empty `keymap.lisp`, and reran `(launch)`. Now I see that not only can kitty move, but that `~/.config/shirakumo/save-the-farm/keymap.lisp` has been automatically recreated and contains my bindings.

So while the original thing I thought was a bug wasn't, perhaps there is some strange interaction still lingering if that "global" `keymap.lisp` is empty.
